math/minuit2/inc/Minuit2/GradientCalculator.h

```diff
@@ -35,6 +35,17 @@ class GradientCalculator {
 
    virtual bool G2(const MinimumParameters &, MnAlgebraicVector &) const { return false;}
 
+   /**
+    * Enable parallelization of gradient calculation using OpenMP.
+    * This is different from the default parallel mechanism elsewhere (IMT, threads, TBB, ...).
+    * It can only be used to minimise thread-safe functions in Minuit2.
+    * \param doParallel true to enable, false to disable
+    * \note If OPENMP is not available, this function has no effect
+    */
+   static void DoParallelOMP(bool doParallel = true) { fDoParallelOMP = doParallel; }
+
+protected:
```

Comment:
```suggestion
private:
```
I don't think this needs to be protected

Reply:
I made that protected because this variable need to be accessible to the child class `Numerical2PGradientCalculator` where the omp is actually implemented.

Reply:
Ah yes! Thanks.